arangodb-net-standard/TransactionApi/Models/PostTransactionResponse.cs

```diff
@@ -11,6 +11,12 @@ public class PostTransactionResponse<T>
         /// <summary>
         /// Whether the request resulted in error.
```

Comment:
Maybe tweak the summary as well? e.g. "Whether the request resulted in error (false in this case)."

Reply:
Possibly, but the actual summary for what this property does at the moment is "Constant: False", which felt a little on the nose.

Reply:
Fair. It was to add some extra obvious, there are a bunch of model that have this `Error` property and a summary saying "false in this case". But here there is the the remarks, so it's fine.